internal/services/controller/controller.go

```diff
@@ -727,15 +758,6 @@ func createEventInformer(f informers.SharedInformerFactory, v version.Interface,
 	})
 }
 
-func fetchAPIResourceLists(client discovery.DiscoveryInterface, log logrus.FieldLogger) []*metav1.APIResourceList {
-	_, apiResourceLists, err := client.ServerGroupsAndResources()
-	if err != nil {
-		log.Warnf("Error when getting server resources: %v", err.Error())
-		return nil
-	}
-	return apiResourceLists
-}
-
```

Comment:
question: what's the reason we deleted this function?

I remember seeing a linter complaining this unexported function was not used anywhere. Is this the reason?

Reply:
Yeh, just cleanup of unused code